coalib/parsing/LineParser.py

```diff
@@ -36,7 +37,8 @@ def __init__(self,
             {'[': ']'} if section_name_surroundings is None
             else section_name_surroundings)
 
-        self.key_value_delimiters = key_value_delimiters
```

Comment:
hm I'm not sure, our parsing stuff is pretty internal - is there really any chance of anyone outside this repo using this? Especially this low level utility.